settings/ci.mdx

```diff
@@ -8,72 +8,365 @@ icon: "circle-check"
 
 ## Installation
 
-To begin, you will need to have followed the steps on the [GitHub](/settings/github) page.
+To begin, follow the steps on the [GitHub](/settings/github) page.
 
-For GitHub Apps, you can choose to only give permissions to a single repository.
-We highly recommend you do so as we only need access to the repository where
-your docs are hosted.
+For GitHub Apps, you can choose to give permissions to only a single repository. We highly recommend you do so as we only need access to the repository where your docs are hosted.
 
 ## Configuration
 
-You can configure the CI checks enabled for a deployment on the Mintlify dashboard by navigating to the 'Add-Ons' tab. There you can enable or disable the checks you'd like to run.
+Configure the CI checks enabled for a deployment by navigating to the [Add-ons](https://dashboard.mintlify.com/products/addons) page of your dashboard. Enable the checks that you want to run.
 
 When enabling checks, you can choose to run them at a `Warning` or `Blocking` level.
 
-<Note>
+- A `Warning` level check will never provide a failure status, even if there is an error or suggestions.
+- A `Blocking` level check will provide a failure status if not passed, or changes are suggested.
 
+## Available CI checks
 
-A `Blocking` level check will provide a failure status if not passed, or changes are suggested.<br/>
+### Broken links
 
-A `Warning` level check will never provide a failure status, even if there is an error or suggestions.
+Similar to how the [CLI link checker](/settings/broken-links#broken-links) works on your local machine, we automatically check your docs for broken links.
 
-</Note>
+To see the results of this check, visit GitHub's check results page for a specific commit.
 
-## When Do They Run?
+### Vale
 
-CI checks are configured to run on commits to your configured deployment branch, or on pull requests against that branch.
+[Vale](https://vale.sh/) is an open-source rule-based prose linter which supports a range of document types, including Markdown and MDX.
 
-## Available CI Checks
+Mintlify supports automatically running Vale in a CI check and displaying the results as a check status.
 
-### Broken Links
+#### Configuration
+If you have a `.vale.ini` file in the root content directory for your deployment, we automatically use that configuration file. We also automatically use any configuration files in your specified `stylesPath`.
 
-Similarly to how the [CLI link checker](/settings/broken-links#broken-links) works on your local machine, we will automatically check your docs for broken links.
-To see the results of this check, you can visit GitHub's check results page for a specific commit.
+If you do not have a Vale config file, the default configuration automatically loads.
 
-### Vale
+````mdx Default vale.ini configuration expandable
```

Comment:
This `expandable` tag doesn't seem to be working locally. Not sure if this is supported in our codeblocks.

We could probably do this with an Accordion?

Reply:
confirmed Dean also has the code blocks rendering as expected, so going to mark these as resolved